walkthroughs/week-5-tdd/intro/src/test/java/com/google/sps/GreeterTest.java

```diff
@@ -30,4 +30,14 @@ public void testGreeting() {
 
     Assert.assertEquals("Hello Ada", greeting);
   }
+
+  @Test
+  public void testGreetingTrimsWhitespace() {
```

Comment:
note: for junit4 tests, the convention is to not have the 'test' prefix for the method name. A good pattern to follow is <test_method_name>_<condition>_<what's expected>, for example greet_nameWithWhitespace_trimsWhitespace so it's immediately clear what the test is asserting

Reply:
Thanks

Reply:
Looks like the name hasn't changed